language-server/src/main/java/org/sonarlint/languageserver/SonarLintLanguageServer.java

```diff
@@ -129,6 +139,7 @@
   private StandaloneSonarLintEngine engine;
   @Nullable
   private URI workspaceRoot;
+  private final List<URI> workspaceFolders = new ArrayList<>();
```

Comment:
I would merge workspaceRoot and workspaceFolders together. See https://github.com/eclipse/eclipse.jdt.ls/blob/41679e9b089b32a60272a9dc1c1273bd5a2e65fa/org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/handlers/InitHandler.java#L88

Reply:
Good idea, will do.